app/src/store/playgroundStore.tsx

```diff
@@ -19,10 +33,17 @@ export type PlaygroundTemplate =
   | PlaygroundChatTemplate
   | PlaygroundTextCompletionTemplate;
 
+export const CHAT_MESSAGE_ROLES = [
+  "user",
+  "assistant",
```

Comment:
I changed this to be 'ai' to not be openai specific. Maybe a bad call but we will have to normalize this I think so in reality we don't want the zod parsing to depend on the store enum anyway